### PR TITLE
[MRG] Tweaks for template to Kwik conversion script

### LINKED ISSUE #301: Optimize waveform creation in store
load spike after spike instead of cluster after cluster (to preserve IO contiguity)

### PR DESCRIPTION
**Pull request by rossant**
Minor fixes for template matching conversion script.

Closes #301

### CODE REVIEW

**Comment by rossant** [comment]
status: awaiting Pierre's feedback

**Comment by rossant** [comment]
ready to go @nippoo

**Comment by rossant** [comment]
@nippoo  **not ready for merge actually**

**Comment by rossant** [comment]
ready to merge @nippoo

File: phy/plot/waveforms.py [hunk 1]
```diff
@@ -308,6 +310,10 @@ def set_data(self,
         self.visual.channel_positions = channel_positions
         self.visual.channel_order = channel_order
 
+        # Extra parameters.
+        for k, v in kwargs.items():
+            setattr(self, k, v)
```
Comment (nippoo) [inline phy/plot/waveforms.py]: was ist das?

Comment (nippoo) [inline phy/plot/waveforms.py]: oh is this R-L clustering distance / density?

Comment (nippoo) [inline phy/plot/waveforms.py]: could they not be extra features?

Comment (rossant) [inline phy/plot/waveforms.py]: no, it's just things like `alpha` etc that will "just work" with `set_data()`

**Pull request merged**